### DISCUSSION TITLE
Why create new s_defaultSimpleConverters and s_defaultFactoryConverters each time when calling method GetConverter of JsonSerializerOptions?

### DISCUSSION THREAD

**Discussion by dczuo** [dotnet/runtime#63260] [Q&A]
https://github.com/dotnet/runtime/blob/b42188a8143f3c7971a7ab1c735e31d8349e7991/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.Converters.cs#L31

**Comment by huoyaoyuan** [comment]
@dotnet/area-system-text-json I had the same question when I viewed the code.
